src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
Oh sorry. I am kind of tired. Maybe I dragged `Point` back into this randomly.

That said it prompted me to talk about our situation in the slack channel, and joelq mentioned making an opaque type. I inferred that this could be implemented as `type Point = Floats Float Float | Ints Int Int` and then we expose functions `Canvas.fromInts` or `fromFloats` or something. That would be more complexity than I think is necessary, but I do have a new idea: lets make a module called `Canvas.Point` that includes a variety of helper functions. That way we can do `Point.fromInts : Int -> Int -> Point`.